app/shared/models/local_artifact_provider.rb

```diff
@@ -15,24 +15,21 @@ def default_root_path
       end
     end
 
-    # @return [Pathname]
```

Comment:
is that change related?

Reply:
Yeah, I combined them. Will pull this back out though